test/Microsoft.AspNetCore.Localization.FunctionalTests/TestRunner.cs

```diff
@@ -14,6 +14,8 @@ namespace Microsoft.AspNetCore.Localization.FunctionalTests
 {
     public class TestRunner
     {
+        private const string ApplicationBasePath = "http://localhost:0";
```

Comment:
I think you should merge this to unflake the build but looking at the history of this comment https://github.com/aspnet/Performance/blob/fa204e1b1878b2214a1f34bee29d0f00af41186a/src/Benchmarks.Utility/Helpers/IISTestManager.cs#L43

It sounds like this is a temporary workaround that was never de-temporaried - is there another item that tracks this?